consensus/types/src/config_and_preset.rs

```diff
@@ -53,8 +52,11 @@ impl ConfigAndPreset {
         let deneb_preset = DenebPreset::from_chain_spec::<E>(spec);
         let extra_fields = get_extra_fields(spec);
 
+        // The latest stable fork is used if the `fork_name` is not specified.
+        let latest_stable_fork = ForkName::latest_stable();
```

Comment:
I think this function was originally intended for testing purposes and I'm not sure if we want the timing of changing this to impact the behaviour of the client, unless we re-purpose it.

Could we use `spec. is_fulu_scheduled` instead, and else fallback to Electra? and we could probably remove the deneb `else` branch below?